packages/graphql-server/package.json

```diff
@@ -9,35 +9,57 @@
   "private": true,
   "scripts": {
     "build:prod": "cross-env NODE_ENV=production rimraf dist && tsc --p tsconfig.json",
+    "build:serverless": "serverless webpack --out dist --stage dev",
     "start:dev": "cross-env REAPIT_ENV=LOCAL nodemon",
     "start:prod": "node ./dist/app.js",
     "test:ci": "cross-env TZ=UTC jest --ci --colors --coverage --silent --forceExit",
-    "test:dev": "cross-env TZ=UTC jest --watch --verbose"
+    "test:dev": "cross-env TZ=UTC jest --watch --verbose",
+    "release:dev": "serverless deploy --stage dev",
+    "release:prod": "node ../../scripts/release/release-serverless.js graphql-server"
   },
   "dependencies": {
     "@reapit/cognito-auth": "2.0.19",
-    "@reapit/elements": "0.5.38",
     "apollo-server": "^2.9.16",
+    "apollo-server-lambda": "^2.10.1",
+    "async": "^3.2.0",
+    "color-convert": "^2.0.1",
+    "color-name": "^1.1.4",
+    "colors": "^1.4.0",
     "dataloader": "^2.0.0",
     "express": "^4.17.1",
     "graphql": "^14.5.8",
     "graphql-depth-limit": "^1.1.0",
     "graphql-import": "^0.7.1",
     "graphql-type-json": "^0.3.1",
+    "is-stream": "^2.0.0",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
Are all of these dependencies or do some really live in dev?

Also, we already have iso-fetch right?

Reply:
Am fine with us hoisting the dependencies because it's quite a special service however, I think I copied a lot of them to the root so there is probably some duplication where we can now remove stuff from the root package json where it is only used in GQL server

Reply:
Not a big deal, just some housekeeping!

Reply:
@willmcvay let me try to remove it and use from the root packages

Reply:
let me try to remote isomophic fetch and use from root package.json